### DISCUSSION TITLE
Amending citations when contributor family name has a space in it

### DISCUSSION THREAD

**Discussion by brettbligh** [knowledgefutures/pubpub#1865] [Support]
Hi,

I'm dealing with a situation where I am manually adding a Pub contributor who has a surname with two words separated by a space. Let's imagine that the contributor I am adding is called "Mohammed Al Maktoum". That's actually the prime minister of the UAE but the real case I'm dealing with has a similar name format.

The citation generated by the "cite" button on the Pub should be: "Al Maktoum, M. (2022)...."

But instead it is more like this, which is incorrect: "Maktoum, M. A. (2022)....."

Is there any way to correct this? Obviously there is only one name field for each contributor, so you can't specify the family name separately. I tried adding a non-breaking space character to replace the space but that didn't change the citation that was generated.

Thanks in advance for any help you can provide!

Brett

**Comment by kyledenlinger** [comment]
This issue also applies to corporate authors. 

Here a fake corporate author called "Center for Media Studies" is treated like a personal name:

<img width="402" alt="image" src="https://user-images.githubusercontent.com/28068891/162788023-fe091b66-d5c4-4e6f-b0a1-63cf314b93ca.png">

**Comment by gabestein** [comment]
Thanks both! We had a nice discussion a week ago about updating our contributor model to better account for both these cases (as well as some other improvements like multiple affiliations tied to ROR IDs, etc). I'm not sure exactly when we'll have updates, but we will be chatting about how to address them starting next week.
